pythonx/netranger/netranger.py

```diff
@@ -420,6 +420,7 @@ def abbrev_cwd(self, width):
                 return '/'.join(sp).ljust(width)
             else:
                 total += len(sp[i]) - 1
+        return res[-1 * width:]
```

Comment:
I suppose we never reach this point. Did you encounter any issue or did you see lsp warning? If it's the later, maybe it's ok to just return res?